pkg/build/api/types.go

```diff
@@ -59,6 +59,10 @@ type BuildSpec struct {
 	// Compute resource requirements to execute the build
 	Resources kapi.ResourceRequirements
 
+	// PostCommit is a build hook executed after the build output image is
+	// committed, before it is pushed to a registry.
+	PostCommit BuildPostCommitSpec
```

Comment:
I would recommend we just use Post, and only have one of them ever.  The
alignment with deployments matters to me.

On Fri, Jan 22, 2016 at 11:14 AM, Ionut Palade <notifications@github.com>
wrote:

> In pkg/build/api/types.go
> <https://github.com/openshift/origin/pull/6715#discussion_r50555320>:
>
> > @@ -59,6 +59,10 @@ type BuildSpec struct {
> >  	// Compute resource requirements to execute the build
> >  	Resources kapi.ResourceRequirements
> >
> > +	// PostCommit is a build hook executed after the build output image is
> > +	// committed, before it is pushed to a registry.
> > +	PostCommit BuildPostCommitSpec
>
> strictly naming wise as i remember it's trying to outline more the fact
> that it takes place between the commit and before the push.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/6715/files#r50555320>.
>